bin/run-dev

```diff
@@ -54,6 +54,13 @@ docker-compose --profile "${cloud_provider}" \
 # Wait until the emulator is running.
 "bin/${emulator}/wait"
 
+# Start a warmup request loop so that when the subsequent server startup is ready, we trigger lazy
+# loading its edit-refresh resources so save developer time.
+for i in {1..60}; do
+  curl --output /dev/null --silent --fail http://localhost:9000/loginForm && break
+  sleep 5
```

Comment:
suggestion (non-blocking): there's a potential annoyance here where this continues to run backgrounded for 5 minutes if it doesn't succeed. it's a pretty minor thing, but i'm imagining a situation where someone is closely watching the logs for debugging or something and one or more of these are still running? not the biggest of deals. one way to avoid it though could be to only make the request once and only make it once the server is listening on the port. it'd be a little cleaner i think:

```bash
function trigger_initial_server_compile() {
  until lsof -i | grep -q "localhost:9000" > /dev/null; do
    sleep 1
  done

  curl --output /dev/null --silent http://localhost:9000/loginForm
}

trigger_initial_server_compile &
```

Reply:
I'm unfamiliar with lsof but that until loop doesn't work.  port 9000 doesn't show up in the lsof -i output with a running server.

I tried to make the request better for the logs by loading a more obvious path but it was a 404 which didn't end the loop :D

Reply:
I'll put it in as is as I think it's better despite the potential annoyance. Feel free to iterate on it subsequently.

Reply:
> I'm unfamiliar with lsof but that until loop doesn't work. port 9000 doesn't show up in the lsof -i output with a running server.

oh hmm, WOMM. my guess the macOS version has different behavior than the linux version